.goreleaser.yaml

```diff
@@ -79,9 +77,18 @@ release:
   github:
     owner: filecoin-project
     name: lotus
-  prerelease: auto
-  name_template: "v{{.Version}}"
+  draft: "{{ .Env.IS_DRAFT }}"
+  header: "{{ .Env.HEADER }}"
+  make_latest: "${{ .Env.IS_LATEST }}"
+  name_template: "{{ .Env.VERSION }}"
+  prerelease: "${{ .Env.VERSION | strings.hasSuffix '-rc' }}"
+  replace_existing_artifacts: true
+  # TODO(p0): Test if not replacing existing drafts updates them instead of failing the release
```

Comment:
Are these p0 todos items that need to be handled before merge?  If so, maybe pull out as PR checklist items or self-comment so it's clear there are unresolved comments?

Reply:
Yes, that was exactly the idea. I think I've resolved all of them now. Apart from this one, but that's not a p0 because we don't create drafts at all in this iteration. I'll update that in a sec.